packages/desktop-client/src/components/Titlebar.tsx

```diff
@@ -177,9 +177,7 @@ export function SyncButton({ style, isMobile = false }: SyncButtonProps) {
     <>
       <KeyHandlers
         keys={{
-          'ctrl+s': () => {
-            sync();
-          },
+          'ctrl+s': sync,
```

Comment:
Turns out KeyHandler's props doesn't like this, so I've reverted this in 81446614a0c3f5c945d96d7d3cfbe37e25d64af7